mod/config/pkg/spec/special_cases.go

```diff
@@ -31,8 +29,10 @@ const (
 	BArtioValRoot = "0x9147586693b6e8faa837715c0f3071c2000045b54233901c2e7871b15872bc43"
 )
 
-const ( // Planned hard-fork upgrades on boonet.
+// Planned hard-fork upgrades on boonet.
+const (
 	BoonetFork1Height uint64 = 69420
 
-	BoonetFork2Height uint64 = math.MaxUint64
+	// TODO: modify this to be the actual fork height. Avoid overflow.
+	BoonetFork2Height uint64 = 99999999999999
```

Comment:
I'll check the rest of the code, but I am surprised we ever overflow.
Like I don't expect much to be done with fork than comparison.
In UTs we can always use a network where fork is not active to be sure we're safe

Reply:
The overflow was happening in tests. Because simulating the slot after the fork upgrade

Reply:
We just have to be careful that we don't accidentally activate this fork in tests. I think the value you pick is fine